bot/utils/__init__.py

```diff
@@ -13,6 +22,50 @@ class CogABCMeta(CogMeta, ABCMeta):
     pass
 
 
+async def send_to_paste_service(http_session: ClientSession, contents: str, *, extension: str = "") -> Optional[str]:
+    """
+    Upload `contents` to the paste service.
+
+    `http_session` should be the current running ClientSession from aiohttp
+    `extension` is added to the output URL
+
+    When an error occurs, `None` is returned, otherwise the generated URL with the suffix.
+    """
```

Comment:
I don't understand why this code should be in `__init__.py`. Let's move this to an appropriate file instead.

Reply:
I didn't feel a need to create a new file for a simple function like this. Are you suggesting for example a path like `bot/utils/paste/send_to_paste_service` ?

Reply:
We shouldn't be defining functions in the `__init__.py` file at all, it's an antipattern. I'm not suggesting that it needs to have a whole file just for itself, it'd be fine for it to share a file with other similar functions. `paste.py` would be okay, but even something more generic like `services.py` would be fine by me. Then we can say oh okay, here are helpers for interacting with external services.

But my primary gripe is just that I don't think anything like this should ever live inside an init file.